cmd/cliutils/globusTransfer.go

```diff
@@ -0,0 +1,42 @@
+package cliutils
+
+import (
+	"log"
+	"strings"
+)
+
+func GlobusTransfer(params TransferParams) (archivable bool, err error) {
+	// === collecting params. ===
+	globusClient := params.GlobusClient
+
+	fileList := params.Filelist
+	isSymlinkList := params.IsSymlinkList
+	srcCollection := params.SrcCollection
+	srcPrefixPath := params.SrcPrefixPath
+	dsSourceFolder := params.DatasetSourceFolder
+
+	destCollection := params.DestCollection
+	destPrefixPath := params.DestPrefixPath
+	datasetId := params.DatasetId
+
+	archivable = false // the dataset is never archivable after a globus transfer request immediately
+	destFolder := destPrefixPath + "/archive/" + strings.Split(datasetId, "/")[1] + dsSourceFolder
+
+	for i := range fileList {
+		fileList[i] = srcPrefixPath + "/" + fileList[i]
```

Comment:
This doesn't seem platform-independent. Shouldn't it use `filepath.Join` and `filepath.ToSlash` to convert to valid linux paths?

Reply:
Globus always uses forward slashes, so I actually went out of my way to do it this way. It also means that the filelist produced by a previous step might not work properly when run under Windows. That could be handled by a separate PR I feel like or I could add a rudamentary "if os == windows, replace '\' with '/'"

Reply:
Source: https://docs.globus.org/globus-connect-personal/install/windows/ (the first NOTE section)

Reply:
I think that's what filepath.ToSlash does, except it would also apply to srcPrefixPath.

Reply:
Yeah, I checked the exact descriptions of "fromSlash" and "toSlash" and that seems to be the case.